euth/ideas/urls.py

```diff
@@ -1,10 +1,11 @@
 from django.conf.urls import url
+from django.contrib.auth.decorators import login_required
 
 from . import views
 
 urlpatterns = [
     url(r'create/module/(?P<slug>[-\w_]+)/$',
-        views.IdeaCreateView.as_view(), name='idea-create'),
+        login_required(views.IdeaCreateView.as_view()), name='idea-create'),
```

Comment:
Instead of wrapping the view in URLconf you can use [django-braces' `LoginRequiredMixin`](https://django-braces.readthedocs.io/en/latest/access.html#loginrequiredmixin) to require a login. This is useful if you reuse the view somewhere else because the view itself will require the login and not just the instance used in URLconf.